src/elements/content-sidebar/versions/__tests__/__snapshots__/VersionsItem-test.js.snap

```diff
@@ -32,6 +32,7 @@ exports[`elements/content-sidebar/versions/VersionsItem render should show actio
           values={
             Object {
               "name": "Test User",
+              "versionPromoted": undefined,
```

Comment:
Should these be in the snap / dom if they are undefined?

Reply:
Yeah, since I'm not filtering them out of the `values` object. It should be functionally equivalent, though.